packages/blade/src/components/Button/BaseButton/buttonTokens.ts

```diff
@@ -9,52 +8,29 @@ import type { BaseButtonProps } from './BaseButton';
 export type ButtonMinHeight = 28 | 32 | 36 | 48;
 
 export type ButtonTypography = {
-  [Key in TypographyPlatforms]: {
-    fonts: {
-      size: Record<
-        NonNullable<BaseButtonProps['size']>,
-        keyof ThemeTokens['typography'][Key]['fonts']['size']
-      >;
-    };
-    lineHeights: Record<
-      NonNullable<BaseButtonProps['size']>,
-      keyof ThemeTokens['typography'][Key]['lineHeights']
-    >;
+  fonts: {
+    size: Record<NonNullable<BaseButtonProps['size']>, keyof Theme['typography']['fonts']['size']>;
```

Comment:
**nit:** 
Just clean this up a bit. 

```ts
type ButtonSizes = NonNullable<BaseButtonProps['size']>;
// replace all repetitions
```

Reply:
why so? we still do need to have this right?
```
fonts: {
    size: {
      xsmall: 75,
      small: 75,
      medium: 100,
      large: 200,
    },
  },
}
```

Reply:
It doesn't change anything. Just we are repeating it a lot. 


<img width="745" alt="Screenshot 2022-07-26 at 4 06 45 PM" src="https://user-images.githubusercontent.com/35374649/180986668-0f57daa2-e0ee-4eb5-9472-c3f4af74789f.png">